config/config.kafka.extended.hocon

```diff
@@ -1,4 +1,9 @@
 {
+  # Full license text available in LICENSE.md
+  license {
+    accept = true
+  }
```

Comment:
This new section stands out as being in a different style from the rest of the file.

To be in-keeping with the rest of the file, it would look like this:

```
"license": {
  "accept": true
}
```

....which is what you've done in the `*.minimal.hocon` examples.